types/share_splitting.go

```diff
@@ -110,15 +110,19 @@ func getNextChunk(rawDatas [][]byte, outerIndex int, innerIndex int, width int)
 	return rawData, outerIndex, innerIndex, startIndex
 }
 
-func GenerateTailPaddingShares(n int, shareWidth int) NamespacedShares {
+// tail is filler for all tail padded shares
+// it is allocated once and used everywhere
+var tail = append(
```

Comment:
nit: maybe a slightly more descriptive name? it's not really necessary from a documentation standpoint, but it is a package var

Reply:
Would you mind suggesting the name, as a native English speaker?

Reply:
tailPaddingShare? just to match TailPaddingNamespace

Reply:
tailPaddingShare is good IMO. I know Hlib prefers shorter var names but in this case the (global) variable should have a descriptive name. A shorter name would be OK, if the var was in the scope of the function itself.